test/unit/puppetlabs/services/master/master_core_test.clj

```diff
@@ -24,11 +24,12 @@
         app         (build-ring-handler handler "1.2.3")
         request     (partial app-request app)]
     (is (= 200 (:status (request "/v3/environments"))))
+    (is (= 200 (:status (request "/v3/catalog/bar?environment=environment1234"))))
+    (is (= 200 (:status (request :post "/v3/catalog/bar?environment=environment1234"))))
```

Comment:
Should this one mock putting the environment parameter in the request body instead of just appending it as a query string at the end of the URL so that it would end up being run more like how a real catalog request would do it?